Altis_Life.Altis/core/functions/fn_keyHandler.sqf

```diff
@@ -44,12 +44,21 @@ if (!(count (actionKeys "User10") isEqualTo 0) && {(inputAction "User10" > 0)})
     true;
 };
 
-if (life_container_active) then {
-    switch (_code) do {
-        //space key
-        case 57: {
-            [] spawn life_fnc_placestorage;
+if (life_container_active) exitwith {
+    //ignore movement actions
+    _allowedMoves = ["MoveForward", "MoveBack", "TurnLeft", "TurnRight", "MoveFastForward", "MoveSlowForward", "turbo", "TurboToggle", "MoveLeft", "MoveRight", "WalkRunTemp", "WalkRunToggle", "AdjustUp", "AdjustDown", "AdjustLeft", "AdjustRight", "Stand", "Crouch", "Prone", "MoveUp", "MoveDown", "LeanLeft", "LeanLeftToggle", "LeanRight", "LeanRightToggle"];
+    if (({_code in (actionKeys _x)} count _allowedMoves) > 0) exitwith {
+        false;
```

Comment:
Use breakOut here instead, as this will only exit out of the scope its in.

false breakOut "main";

Be sure to add a scopename to the top of the file.

Reply:
the whole thing is already in an exitwith, that one is just to prevent it from running the rest of the code